social-network/src/ArticleCard.js

```diff
@@ -0,0 +1,39 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+
+class ArticleCard extends Component {
+    static propTypes = {
+        img: PropTypes.string.isRequired,
+        title: PropTypes.string.isRequired,
+        description: PropTypes.string.isRequired,
+        authorImg: PropTypes.string.isRequired,
+        author: PropTypes.string.isRequired,
+        date: PropTypes.string.isRequired,
+        readTime: PropTypes.string.isRequired,
+    }
+
+    render() {
+        const months = ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sept', 'Oct', 'Nov', 'Dec'];
+        const monthNum = parseInt(this.props.date.substring(5, 7));
+        const month = months[monthNum - 1];
+        const day = this.props.date.substring(8, 10).replace(/^0+/, '');
+
```

Comment:
You may also want to look into how to use date to format your strings for you. 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleDateString